repo_checker.py

```diff
@@ -166,7 +166,8 @@ def prepare_review(self):
                 if status and str(status['overall_state']) == 'failed':
                     # Exception to the rule is openQA only in failed state,
                     # Broken packages so not just openQA.
-                    openQA_only = (len(status['broken_packages']) == 0)
+                    broken_packages = [a for a in status['broken_packages'] if not a['package'].startswith('000') ]
```

Comment:
This is really ugly, at least toss a comment in there so future generations know why this code was so bludgeoned. Seems more like it should be ignored on API end since likely useless to display in dashboard as failure as well.

Reply:
Meh, I guess the intention is the check should block staging, but you want repo checker to proceed. At minimum a comment.